tools/pythonpkg/src/pyrelation.cpp

```diff
@@ -198,23 +202,23 @@ unique_ptr<DuckDBPyRelation> DuckDBPyRelation::AliasDF(py::object df, const stri
 }
 
 unique_ptr<DuckDBPyRelation> DuckDBPyRelation::Filter(const string &expr) {
-	return make_unique<DuckDBPyRelation>(rel->Filter(expr));
+	return make_unique<DuckDBPyRelation>(rel->Filter(expr), map_function);
```

Comment:
I think this should not be in every single constructor. Can't we keep a reference to the function in the map relation only?